src/cli/serve.rs

```diff
@@ -88,7 +90,10 @@ fn show_start_message(bind_address: IpAddr, port: u16, color: ColorChoice) -> io
     write!(&mut buffer, "Visit ")?;
 
     buffer.set_color(&green)?;
-    write!(&mut buffer, "http://localhost:{}/", port)?;
+    write!(&mut buffer, "http://{}:", address_string)?;
+
+    buffer.set_color(&green)?;
+    write!(&mut buffer, "{}/", port)?;
```

Comment:
Let's merge these calls together. You can pass multiple arguments into the same format string, something like this should work:

```rust
write!(&mut buffer, "http://{}:{}/", address_string, port)?;
```